include/cantera/numerics/FuncEval.h

```diff
@@ -94,6 +129,9 @@ class FuncEval
 
     //! Errors occurring during function evaluations
     std::vector<std::string> m_errors;
+
+    //! Pointer to preconditioner - nullptr unless otherwise set
+    PreconditionerBase *m_preconditioner = nullptr;
```

Comment:
Put the modifier (`*` or `&`) next to the type name, rather than the variable name.
```suggestion
    PreconditionerBase* m_preconditioner = nullptr;
```